easyaccess/easyaccess.py

```diff
@@ -1370,49 +1362,25 @@ def do_set_password(self, arg):
                 print(sys.exc_info())
 
 
+
     def do_refresh_metadata_cache(self, arg):
         """
         DB:Refreshes meta data cache for auto-completion of table
         names and column names .
         """
-
-        # Meta data access: With the two linked databases, accessing the
-        # "truth" via fgetmetadata has become very slow.
-        # what it returns is a function of each users's permissions, and their
-        # "mydb". so yet another level of caching is needed. Ta loads a table
-        # called fgottenmetadata in the user's mydb. It refreshes on command
-        # or on timeout (checked at startup).
-
-        # get last update
         verb = True
-        if arg == 'quiet': verb = False
-        query_time = "select created from dba_objects where object_name = \'FGOTTENMETADATA\' and owner =\'%s\'  " % (
-            self.user.upper())
         try:
-            qt = self.cur.execute(query_time)
-            last = qt.fetchall()
-            now = datetime.datetime.now()
-            diff = abs(now - last[0][0]).seconds / (3600.)
-            if verb: print('Updated %.2f hours ago' % diff)
-        except:
-            pass
-        try:
-            query = "DROP TABLE FGOTTENMETADATA"
```

Comment:
This should have been `DROP TABLE FGOTTENMETADATA PURGE`. It looks like it has been removed completely, but that makes me wonder whether `FGOTTENMETADATA` is ever updated automatically.